Project.toml

```diff
@@ -0,0 +1,26 @@
+name = "ImageShow"
+uuid = "4e3cecfd-b093-5904-9786-8bbb286a6a31"
+version = "0.2.0"
+
+[deps]
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
+Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+
+[compat]
+ColorTypes = ">= 0.7.4"
+julia = ">= 1.0"
+
+[extras]
+ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
+PaddedViews = "5432bcbf-9aad-5242-b902-cca2824c8663"
+QuartzImageIO = "dca85d43-d64c-5e67-8c65-017450d5d020"
```

Comment:
I am guessing the `@osx` marcos etc are gone? should we maybe just stcik with ImageMagick in this case?

Reply:
According to https://github.com/JuliaLang/Pkg.jl/issues/165 this seems not implemented yet.